src/App.tsx

```diff
@@ -8,37 +8,107 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { User } from './types/User';
+import { getUsers } from './api/users';
+import { Post } from './types/Post';
+import { getPosts } from './api/posts';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [selectedPost, setSelectedPost] = useState<Post | null>(null);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const getSelectedUser = (user: User) => {
```

Comment:
Consider renaming the function 'getSelectedUser' to 'handleSelectUser' for clarity and consistency with common naming conventions.

Reply:
Thanks, good to know.